packages/splits-sdk-react/src/hooks/splitV2.ts

```diff
@@ -57,7 +75,7 @@ export const useCreateSplitV2 = (): {
     [splitsClient],
   )
 
-  return { createSplit, status, txHash, error }
+  return { createSplit, splitAddress, status, txHash, error }
```

Comment:
its fine imo since its part of an object and its optional, one question, why are we adding this now, did we get a request for it? Its a good addition though just curious

Reply:
yea a user was having trouble getting the split address. i realized we make it very easy from the core sdk, but it's pretty convoluted from the react sdk (need to decode the logs).

there's a long thread in the dev-support channel